setup.py

```diff
@@ -28,14 +28,10 @@
         "numpy",
         "rich",
         "rtree",  # Used by sumolib
-        "filelock",
-        "lz4",
-        "networkx",
-        "opencv-python",
```

Comment:
Glad we're able to get rid of some things.  Did you also audit the list of things in `requirements.txt`?  The more we can get rid of there, the better!

Reply:
I removed the above unused packages and then ran the setup again to see which packages were installed as dependencies. And then updated the `requirements.txt` from the output of `pip freeze`. I can explicitly remove some packages listed in there but not sure if that's going to break things or not.